src/graphics/pipeline.rs

```diff
@@ -84,9 +85,9 @@ fn layout(name: &str, device: &Device) -> wgpu::PipelineLayout {
     })
 }
 
-fn multisample(alpha_to_coverage_enabled: bool) -> wgpu::MultisampleState {
+fn multisample(count: u32, alpha_to_coverage_enabled: bool) -> wgpu::MultisampleState {
     wgpu::MultisampleState {
-        count: MULTISAMPLE_COUNT,
+        count,
         mask: !0,
```

Comment:
Hahaha, what? Not zero? Is that an integer filled with ones?

Sorry, I know it's not new, but I managed to not see it in the previous reviews!

Reply:
I'm also confused by this, here's the passage from the doc :

> Bitmask that restricts the samples of a pixel modified by this pipeline. All samples can be enabled using the value !0

Reply:
I looked it up. This is a bitmask with 64 ones

Reply:
So it's an integer filled with ones, identical to `int i = -1;`. Thanks.